File: 0002. Hollow Rectangle/code.cs
using System;


namespace HollowRectangle
{
    class Program
    {
        
        static void Main(string[] args)
        {
            Console.Write("Enter the number of rows : ");
            int rows = Convert.ToInt32(Console.ReadLine());

            Console.Write("Enter the number of cols : ");
            int cols = Convert.ToInt32(Console.ReadLine());

            for (int i = 1; i <= rows; i++)
            {
                for (int j = 1; j <= cols; j++)
                {
                    if (i == 1 || i == rows || j == 1 || j == cols)
                    {
                        Console.Write("*");
                    }
                    else
                    {
                        Console.Write(" ");
                    }

                }
                Console.Write("\n");
            }
        }
    }
}

File: 0018. Number Pattern #2/code.cpp
#include<iostream>
using namespace std;

int main(){

  int i,j,k;
  int n;

  cout <<"Enter the number of rows : ";
  cin >>n;

  cout << endl;
  for(i=n; i>=1; i--)
  {
    for(j=1; j<=n; j++)
    {
      if(j <= i)
        cout << j;
      else
        cout << "*";
    }
    for(j=n; j>=1; j--)
    {
      if(j <= i)
        cout << j;
      else
        cout << "*";
    } 
    cout << endl;
  }

  cout << endl;
  return 0;

}
